components/brave_new_tab_ui/components/default/braveToday/customize/Context.tsx

```diff
@@ -110,12 +111,17 @@ export const useBraveNews = () => {
 export const useChannels = (options: { subscribedOnly: boolean } = { subscribedOnly: false }) => {
   const { channels } = useBraveNews()
   return useMemo(() => Object.values(channels)
-    .filter(c => c.subscribed || !options.subscribedOnly), [channels, options.subscribedOnly])
+    .filter(c => c.subscribedLocales.length || !options.subscribedOnly), [channels, options.subscribedOnly])
```

Comment:
Is this going to result in displaying all global channels in the initial screen of the Discover section? If so, I think we need to filter to the "current" locale.

Reply:
So this is used by the sidebar to ensure we show channels the user has subscribed to in all locales. Thinking about it though, I think we're already showing all channels in all locales (because the channels system just gets all channels from all publishers).

I think (as the system already works like this) we should fix it in a separate PR, as we can't land Rana's backend changes until this is ready.

Reply:
I've filed this issue:
https://github.com/brave/brave-browser/issues/26385